Lib/_collections_abc.py

```diff
@@ -305,6 +311,11 @@ def __subclasshook__(cls, C):
             return _check_methods(C, "__reversed__", "__iter__")
         return NotImplemented
 
+Reversible.register(dict)
+Reversible.register(dict_keys)
+Reversible.register(dict_values)
+Reversible.register(dict_items)
+
```

Comment:
What is `isinstance({}, Reversible)` output without register?

Since Reversible uses `__subclasshook__`, explicit registering will be redundant.